akka-http-tests/src/test/scala/akka/http/scaladsl/marshalling/MarshallingSpec.scala

```diff
@@ -25,6 +26,8 @@ class MarshallingSpec extends FreeSpec with Matchers with BeforeAndAfterAll with
   implicit val materializer = ActorMaterializer()
   import system.dispatcher
 
+  val marshallingTimeout = 1.second
```

Comment:
Shouldn't these come from the config now/marshalling test utils now?

Reply:
Yes indeed - as these were tests I just set a default timeout but indeed they should come from conf - I've moved around things a bit so they're fetched from conf directly